.github/workflows/pr.yaml

```diff
@@ -0,0 +1,103 @@
+name: PR checks
+on: pull_request
+env:
+  SECRET_KEY: insecure_test_key
+
+jobs:
+  run-image:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build image
+        run: DOCKER_BUILDKIT=1 docker build --tag webteam-releases.canonical.com .
```

Comment:
```suggestion
        run: DOCKER_BUILDKIT=1 docker build --tag webteam.canonical.com .
```

Reply:
Yeah, but what happens if we have more projects for `webteam.canonical.com` ? I assume this is not gonna be just for releases

Reply:
I think for now it's fine to call it like this. We can iterate through his when we face the problem